src/molecules/end/Comparator.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import { Icon } from 'src/utils/const';
+import { Chip, ChipProps } from 'src/molecules/Chip';
+import { layoutStyles } from 'src/utils/styles';
+import { registeredTextStyle, UrbiFontStyles } from 'src/utils/textStyles';
+
+const styles = StyleSheet.create({
+  Wrapper: {
+    height: 72,
+    ...layoutStyles.ColumnJustifyStart,
+    alignItems: 'flex-end',
+    width: '40%',
+  },
+  Title: registeredTextStyle('titleBold' as keyof UrbiFontStyles),
+  Content: { marginTop: 8 },
+  BottomPanel: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    marginTop: 8,
+  },
+  ComparatorBodyMargin: {
+    flexShrink: 1000,
```

Comment:
isn't this supposed to be either `0`, `1`, or `undefined`?

Reply:
Thanks. Commit dd78f75